spec/fixture/two_locations_manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "author": {
+      "name": "Zendesk",
+      "email": "support@zendesk.com"
+    },
+    "location": [
```

Comment:
you're testing a correct manifest? Isn't this manifest supposed to be broken?

Reply:
it's happening to one of our official apps. This manifest is a copy/paste.